charts/kube-prometheus/templates/grafana/dashboards/k8s-cluster-rsrc-use.yaml

```diff
@@ -1,23 +1,23 @@
 {{- /*
-Generated from 'k8s-cluster-rsrc-use' from https://raw.githubusercontent.com/coreos/kube-prometheus/release-0.1/manifests/grafana-dashboardDefinitions.yaml
+Generated from 'k8s-cluster-rsrc-use' from https://raw.githubusercontent.com/kube-prometheus/kube-prometheus/release-0.1/manifests/grafana-dashboardDefinitions.yaml
```

Comment:
```suggestion
Generated from 'k8s-cluster-rsrc-use' from https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/release-0.1/manifests/grafana-dashboardDefinitions.yaml
```

Reply:
Hi @omegas27 thanks for fixing that. I made all 83 changes in one commit: 6356d02 and listed you as a co-author 👍 

**For everyone else following:** this is a non-functional typo fix to commented out CRD URLs 🙂📝

Reply:
This is definitely better than 83 commits :D

Reply:
Repo settings are for PRs to be squashed anyway, but yeah 😅